docs/reference/async-search.asciidoc

```diff
@@ -0,0 +1,20 @@
+[role="xpack"]
+[testenv="basic"]
+[[async-search-intro]]
+== Long running searches
+
+{es} generally allows to quickly search across big amounts of data. There are
+situations where a search executes on many many shards, possibly against
+<<frozen-indices>> and spanning multiple <<modules-remote-clusters>>, for which
```

Comment:
We need to provide an anchor text. Otherwise, these will be capitalized.
```suggestion
<<frozen-indices,frozen indices>> and spanning multiple
<<modules-remote-clusters,remote clusters>>, for which
```